burnman/seismic.py

```diff
@@ -14,38 +21,30 @@ def __init__(self):
 
 
 
-    def evaluate_all_at(self, depth_list):
+    
+    def evaluate(self, vars_list, depth_list):
         """
         Returns the lists of data for a Seismic1DModel for the depths provided
-            
+        
         Parameters
         ----------
+        vars_list : array of variables (in str format) to be evaluated
         depth_list : array of floats
-            Array of depths [m] to evaluate seismic model at.
-            
+        Array of depths [m] to evaluate seismic model at.
```

Comment:
This, and other places: the numpydoc docstring format usually has an indentation in front of the description of a parameter. Not sure whether this affects the sphinx parsing.